src/mcp/server/fastmcp/server.py

```diff
@@ -559,8 +560,16 @@ async def handle_sse(request: Request) -> EventSourceResponse:
             from mcp.server.auth.routes import create_auth_routes
 
             required_scopes = self.settings.auth.required_scopes or []
-
+            
             middleware = [
+                # Add CORS middleware to allow cross-origin requests
+                Middleware(
+                    CORSMiddleware,
+                    allow_origins=["*"],  # Allow any origin
+                    allow_methods=["GET", "POST", "OPTIONS"],
+                    allow_headers=["*"],
+                    allow_credentials=True,
+                ),
```

Comment:
The CORSMiddleare needs to wrap the application, not be included in the middleware parameter.